custodial-copy-reconciler/src/test/scala/uk/gov/nationalarchives/reconciler/DatabaseSpec.scala

```diff
@@ -0,0 +1,305 @@
+package uk.gov.nationalarchives.reconciler
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import doobie.implicits.*
+import doobie.util.Get
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers.*
+import org.scalatest.prop.TableDrivenPropertyChecks.forAll
+import org.scalatest.prop.Tables.Table
+import org.scalatest.prop.{TableFor4, TableFor5}
+import uk.gov.nationalarchives.reconciler.Main.Config
+import uk.gov.nationalarchives.reconciler.{Configuration, Database, OcflCoRow, PreservicaCoRow}
+import uk.gov.nationalarchives.utils.TestUtils.*
+
+import java.net.URI
+import java.nio.file.{Files, Path}
+import java.util.UUID
+
+class DatabaseSpec extends AnyFlatSpec with BeforeAndAfterEach:
+  given Get[UUID] = Get[String].map(UUID.fromString)
+  case class ReconcilerDatabaseUtils() extends DatabaseUtils("test-database") {
+    def createOcflCoRow(
+        coRef: UUID,
+        ioRef: UUID = UUID.randomUUID(),
+        sha256Checksum: Option[String] = None,
+        sha1Checksum: Option[String] = None,
+        md5Checksum: Option[String] = None
+    ): IO[OcflCoRow] =
+      sql"""INSERT INTO ExpectedCosInPS (coRef, ioRef, representationType, generationType, sha256Checksum, sha1Checksum, md5Checksum)
+                 VALUES (${coRef.toString}, ${ioRef.toString}, "Preservation_1", "Original", $sha256Checksum, $sha1Checksum, $md5Checksum)""".update.run
+        .transact(xa)
+        .map(_ => OcflCoRow(coRef, ioRef, "Preservation_1", "Original", sha256Checksum, sha1Checksum, md5Checksum))
+
+    def createPSCoRow(
+        coRef: UUID,
+        ioRef: UUID = UUID.randomUUID(),
+        sha256Checksum: Option[String] = None,
+        sha1Checksum: Option[String] = None,
+        md5Checksum: Option[String] = None
+    ): IO[PreservicaCoRow] =
+      sql"""INSERT INTO ActualCosInPS (coRef, ioRef, generationType, sha256Checksum, sha1Checksum, md5Checksum)
```

Comment:
You'll need to change these to not insert generationType, sha1Checksum and md5Checksum.